### PR TITLE
net-analyzer/wireshark: add 4.4.0, fix handling of release notes

### PR DESCRIPTION
**Pull request by hhoffstaette**
Various changes:
- Supported Lua versions changed
- NEWS is gone
- "Help->Release Notes" never worked ([#939195](https://bugs.gentoo.org/939195))
- Fix inconsistent keywords on -9999

For USE=-doc we install a release-notes.html file that redirects to the corresponding online version.

Closes: https://bugs.gentoo.org/939195

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by hhoffstaette** [comment]
Sad note about g_build_filename/path and path canonicalization: I tried to use [g_canonicalize_filename](https://docs.gtk.org/glib/func.canonicalize_filename.html) but it does absolutely nothing to remove duplicate path elements. :disappointed: 
Therefore we just remove the install_prefix for now. I could not find any place in our cmake setup where we accidentally pass wrong/duplicate values, so I guess this is just broken upstream. Maybe they build other Linux packages with install_prefix=/.

**Comment by hhoffstaette** [comment]
Demoting to draft until the bug in cmake.eclass is resolved.

**Comment by hhoffstaette** [comment]
Removed the patch from -9999 since it has been accepted and will be merged soon.

File: net-analyzer/wireshark/wireshark-9999.ebuild [hunk 1]
```diff
@@ -284,8 +283,14 @@ src_install() {
 	# https://gitlab.com/wireshark/wireshark/-/commit/fe7bfdf6caac9204ab5f34eeba7b0f4a0314d3cd
 	cmake_src_install install-headers
 
+	# prepare Relase Notes redirector if necessary (bug #939195)
+	local relnotes="doc/release-notes.html"
+	if ! use doc; then
+		sed -e "s/#VERSION#/${PV}/g" < "${FILESDIR}/release-notes.html" > ${relnotes} || die
```
Comment (hhoffstaette) [inline net-analyzer/wireshark/wireshark-9999.ebuild:289]: I just realized that this is of course wrong for -9999 :disappointed:

**Comment by hhoffstaette** [comment]
Removed ~ia64 from KEYWORDS.

**Comment by hhoffstaette** [comment]
Also added ~loong to be in line with 4.2.6/9999 in the tree.

**Comment by hhoffstaette** [comment]
Rebased to master@549cdd3ac1db8f to sort out the KEYWORDS inconsistencies.

**Comment by thesamesam** [comment]
Thanks & sorry for the wait. As a followup, would you consider dropping Qt 5 support per Ionen's email (https://public-inbox.gentoo.org/gentoo-dev/ZtR2Z4GkuzkP4ZQl@eversor/T/#u)?

**Pull request closed without merging**